blocks/library/image/block.js

```diff
@@ -136,7 +136,7 @@ class ImageBlock extends Component {
 	}
 
 	getAvailableSizes() {
-		return get( this.props.image, [ 'data', 'media_details', 'sizes' ], {} );
+		return get( this.props.image, [ 'media_details', 'sizes' ], {} );
```

Comment:
Observing that we're still pretty bound to the REST API structure of an entity with the new `core-data` module. This has its upsides (easy state insertion, consistency), but also doesn't help decouple from the REST API.

Reply:
Agreed, I was wondering if we should introduce "models" somehow